UI.tsv

```diff
@@ -421,12 +421,12 @@ UI_20150317_000420	Parts 1
 UI_20150317_000421	Parts 2
 UI_20150317_000422	Parts 3
 UI_20150317_000423	Congratulations! Your level has increased.
-UI_20150317_000424	Items Alarm
+UI_20150317_000424	Item Alarm
 UI_20150317_000425	{@st64}Details{/}
 UI_20150317_000426	How to Play
-UI_20150317_000427	{@st42b}      Move character up, down, left, right{/}
-UI_20150317_000428	{@st42b}      Launch basic attack{/}
-UI_20150317_000429	{@st42b}Character jumps on the spot or while moving{/}
+UI_20150317_000427	{@st42b}          Move once with each of the keys{/}
+UI_20150317_000428	{@st42b}      Use a basic attack{/}
```

Comment:
"Executes" sounds so different. "Use" is better, because the game requires you to *use* a basic attack.